addons/dashboard/2.0.x/dashboard-3.yaml

```diff
@@ -30,7 +29,7 @@ spec:
   chartReference:
     chart: kubernetes-dashboard
     repo: https://kubernetes.github.io/dashboard/
-    version: 2.0.3
+    version: 2.3.0
```

Comment:
Chart version is 2.3.0, is appversion supposed to be?

Reply:
Seems like the new upstream thinks that the appversion is in fact `v2.0.3`: https://github.com/kubernetes/dashboard/blob/master/aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml#L18